example/00-backup-restore-server-config.yaml

```diff
@@ -46,4 +46,4 @@ defragmentationSchedule: "0 0 */3 * *"
 
 compressionConfig:
    enabled: true
-   policy: "flate"
+   policy: "lzw"
```

Comment:
I would suggest the default to be gzip.

Reply:
deafult is still gzip if we comment this configuration . see here [https://github.com/gardener/etcd-backup-restore/pull/293/files#diff-edf8a5224f35b87bf6e0367f2b78e678d442e8329b0143d0b9653b42be7b752bR29](url)